src/playlist.js

```diff
@@ -0,0 +1,66 @@
+var playItem = require('./playitem.js');
+var setupAutoadvance = require('./autoadvance.js');
+
+var playlistMaker = function(player, plist) {
+  var currentIndex = 0;
+  var autoadvanceTimeout = null;
+
+  var playlist = function playlist(list) {
+    if (typeof list === 'number') {
+      return player.playlist.item(list);
```

Comment:
If this was a contrib plugin, I would have *slightly* less issues with adding more methods to the player itself.
However, particularly *this* early in the game, I'd rather go with the less API as possible. We can always add more methods later on if we find they are needed. Taking away methods is always very hard to do.